packages/web-components/src/components/rux-tabs/rux-tabs.tsx

```diff
@@ -55,6 +55,39 @@ export class RuxTabs {
         }
     }
 
+    @Listen('keydown')
+    onKeydown(e: any) {
+        // Get all tabs inside of the tab group and then
+        // filter out disabled tabs since we need to skip those
+        const tabs = this._tabs.filter((tab) => !tab.disabled)
+
+        // Only move the tab if the current focus is in the tab group
+        if (e.target && tabs.includes(e.target)) {
+            const index = tabs.findIndex((tab) => tab === e.target)
+
+            let next
+
+            // If hitting arrow down or arrow right, move to the next radio
```

Comment:
can we update the comments radio -> tabs

Reply:
Sure thing! Put it in the same commit as the new disabled tabs test